modules/slacker-api/src/main/java/com/labs2160/slacker/api/Resource.java

```diff
@@ -5,7 +5,7 @@
  *
  * Examples: XMPP chat server, IMAP email client, SQL database connection
  */
-public interface Resource extends Configurable {
+public interface Resource extends ConfigurableRaw {
```

Comment:
We need a better name/approach for Resource.  Why does a Resource have to have a start() (and shutdown())?